src/server/api/routes/reviews.router.js

```diff
@@ -20,7 +20,7 @@ const reviewsController = require('../controllers/reviews.controller');
  *      5XX:
  *        description: Unexpected error.
  */
-router.get('/:id', (req, res, next) => {
+router.get('/cv/:id', (req, res, next) => {
```

Comment:
Why is `cv` added here?

Reply:
because both the endpoints are calling with ID and if we don't comment this route it's executing first and am getting the data for specific cv review. so,i thought that it will be good to change endpoint. so, no one get confused

Reply:
but this endpoint is supposed the be for getting a specific review. if you want a specific endpoint for maybe it should be a new separate one? now what if someone already was using the endpoint and now you edited it and then the other code would break :)

Reply:
no one is using that endpoint. it only made with mismatching description. so, daniel said let it be and created new issue.